### PR TITLE
Add Tenderly Documentation

### LINKED ISSUE #744: [docs] Tenderly support in documentation
Tenderly is supporting us on two fronts:

- **First one:**

Web3 Gateway (only for Ethereum Boba atm), 
the docs on docs.boba.network need to mention this, similar to Optimism mentioning Alchemy:
image

The public endpoints are:
```
We have partnered with Tenderly. 

http://boba-ethereum.gateway.tenderly.co should work 

or alternatively http://gateway.tenderly.co/public/boba-ethereum

These are public and have ip based rate limiting. You can subscribe to Tenderly if you need more
```

- **Second one** is available on ALL chains (excluding Moonbeam since it's shutting down) - Tenderly development platform:

image

These things need to be mentioned, perhaps with some examples...

### PR DESCRIPTION
**Pull request by wsdt**
## Overview

Add Tenderly documentation. 

Resolves #744 

## Changes

- Add Tenderly RPC endpoints
- Add Tenderly documentation + examples

### CODE REVIEW

File: boba_documentation/developer/network-parameters.md [hunk 1]
```diff
@@ -35,12 +35,12 @@ For **secondary addresses**, such as L2 Tokens, Messengers, and the DAO, please
...
+| Gateway            | [https://gateway.boba.network](https://gateway.boba.network)                                                                                                                                                                                                                                                                                                             |
+| Blockexplorer      | [https://bobascan.com](https://bobascan.com)                                                                                                                                                                                                                                                                                                                             |
+| Websocket          | [wss://ws.mainnet.boba.network](wss://wss.mainnet.boba.network)                                                                                                                                                                                                                                                                                                          |
```
Comment (InoMurko) [inline boba_documentation/developer/network-parameters.md]: Can you try if tenderly does Wss?

File: boba_documentation/developer/network-parameters.md [hunk 2]
```diff
@@ -35,12 +35,12 @@ For **secondary addresses**, such as L2 Tokens, Messengers, and the DAO, please
...
+|--------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Mainnet ChainID    | 288                                                                                                                                                                                                                                                                                                                                                                      |
+| Mainnet RPC Read   | [https://lightning-replica.boba.network](https://lightning-replica.boba.network)                                                                                                                                                                                                                                                                                         |
```
Comment (InoMurko) [inline boba_documentation/developer/network-parameters.md]: These should be removed now

File: boba_documentation/developer/network-parameters.md (hunk 1 above)
Comment (wsdt) [reply boba_documentation/developer/network-parameters.md]: image

looks good yes!

Comment (wsdt) [reply boba_documentation/developer/network-parameters.md]: added

**Review by souradeep-das — APPROVED** [review]
lgtm!

File: boba_documentation/developer/network-parameters.md [hunk 3]
```diff
@@ -35,12 +35,11 @@ For **secondary addresses**, such as L2 Tokens, Messengers, and the DAO, please
...
+| Gateway            | [https://gateway.boba.network](https://gateway.boba.network)                                                                                                                                                                                                                                                                                                             |
+| Blockexplorer      | [https://bobascan.com](https://bobascan.com)                                                                                                                                                                                                                                                                                                                             |
+| Websocket          | [wss://ws.mainnet.boba.network](wss://wss.mainnet.boba.network)<br/><br/>Tenderly: <br/> [wss://boba-ethereum.gateway.tenderly.co](wss://boba-ethereum.gateway.tenderly.co)<br/>[wss://gateway.tenderly.co/public/boba-ethereum](wss://gateway.tenderly.co/public/boba-ethereum)                                                                                         |
```
Comment (InoMurko) [inline boba_documentation/developer/network-parameters.md]: @Boyuan-Chen should we drop wss support from our l2geth? seems like unneded exposure

wss://wss.mainnet.boba.network/
wss://ws.mainnet.boba.network/

File: boba_documentation/developer/tenderly.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+---
+description: Debugging with Tenderly
+---
+
+# What is Tenderly?
+[Tenderly](https://tenderly.co/) is a great platform that supports your Web3 development efforts. It helps developers to build, test, monitor and operate their smart contracts.
+
+## What you can do
+Tenderly is enabled on all active Boba networks.
```
Comment (InoMurko) [inline boba_documentation/developer/tenderly.md]: Tenderly development platform (Web3 Gateway just for Ethereum, for now)

File: boba_documentation/developer/network-parameters.md (hunk 3 above)
Comment (wsdt) [reply boba_documentation/developer/network-parameters.md]: we could add a note: "we recommend the Tenderly gateway for websocket connections"

Comment (boyuan-chen) [reply boba_documentation/developer/network-parameters.md]: > @Boyuan-Chen should we drop wss support from our l2geth? seems like unneded exposure
> 
> wss://wss.mainnet.boba.network/ wss://ws.mainnet.boba.network/

Sounds good!

Comment (InoMurko) [reply boba_documentation/developer/network-parameters.md]: Will make a ticket in the appropriate repo

Comment (InoMurko) [reply boba_documentation/developer/network-parameters.md]: ok @wsdt it's safe to remove `wss://ws.mainnet.boba.network](wss://wss.mainnet.boba.network` from the docs

File: boba_documentation/developer/network-parameters.md [hunk 5]
```diff
@@ -35,12 +35,11 @@ For **secondary addresses**, such as L2 Tokens, Messengers, and the DAO, please
...
+| Mainnet ChainID    | 288                                                                                                                                                                                                                                                                                                                                                                      |
+| Mainnet Write RPCs | [https://mainnet.boba.network](https://mainnet.boba.network)<br/><br/>We have partnered with Tenderly (ip-based rate limiting, subscribe to Tenderly if you need more): <br/>[http://boba-ethereum.gateway.tenderly.co](http://boba-ethereum.gateway.tenderly.co)<br/>[http://gateway.tenderly.co/public/boba-ethereum](http://gateway.tenderly.co/public/boba-ethereum) |
+| Gateway            | [https://gateway.boba.network](https://gateway.boba.network)                                                                                                                                                                                                                                                                                                             |
```
Comment (InoMurko) [inline boba_documentation/developer/network-parameters.md:42]: if it's not too much to ask, all gateway links (for all chains) should point to this URL

Comment (InoMurko) [reply boba_documentation/developer/network-parameters.md:42]: https://github.com/bobanetwork/boba/issues/766

Comment (wsdt) [reply boba_documentation/developer/network-parameters.md:42]: will do!

**Review by InoMurko — APPROVED** [review]

**Pull request merged**